inst/htmlwidgets/datatables.js

```diff
@@ -668,28 +668,30 @@ HTMLWidgets.widget({
     // run the callback function on the table instance
     if (typeof data.callback === 'function') data.callback(table);
 
+    if (data.editable) {
```

Comment:
This PR looks good to me, but I hope to avoid the changes from indentation. You don't need the curly braces since it is a single expression, e.g.

```js
if (data.editable) table.on('dblclick.dt', 'tbody td', function() {
  ...
```